resources/examples/Showtimes/Controllers/Movie.php

```diff
@@ -118,6 +118,7 @@ public function PATCH()
      * @api-uriSegment {/movies/+id} id (integer) - Movie ID
      *
      * @api-contentType application/json
+     * @api-capability DELETE_CONTENT
```

Comment:
Using the same capability `DELETE_CONTENT` as a scope `delete` for an example isn't the best example of a capability. Might want to change the capability name to something else.